### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jvanhoefer**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (yannikschaelte) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
**To reproduce**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Steps to reproduce the behavior:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Desktop (please complete the following information):**
```
Comment (yannikschaelte) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
**Environment**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ - OS: [e.g. iOS]
+ - Version [e.g. 0.2.1]
```
Comment (yannikschaelte) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]:
```suggestion
 - yaml2sbml version (`pip show yaml2sbml`) [e.g. 0.2.1]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ - If relevant: Version of other tools involved [libsbml]
```
Comment (yannikschaelte) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]:
```suggestion
 - If relevant: Versions of other tools involved [libsbml]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (yannikschaelte) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Maybe wrap text that users should delete in [I am a dummy text]?

**Review by yannikschaelte — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (yannikschaelte) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: oh, and python version

**Comment by jvanhoefer** [comment]
It's funny, that codacy is complaining about parts coming from githubs default text... :D

**Comment by yannikschaelte** [comment]
That is just bad from codacy. I'd either just ignore or disable on codacy if possible.

**Pull request merged**